test/test_models.py

```diff
@@ -74,12 +74,47 @@ def get_available_video_models():
 )
 
 
+# The following contains configuration parameters for all models which are used by
+# the _test_*_model methods.
+_model_params = {
+    'default_classification': {
```

Comment:
nit: instead of declaring a global dict, should we instead declare one dict in each of `_test_classification_model`, `_test_segmentation_model`, etc? This would ease reading the code as relevant info would be where it's used

Reply:
Sounds good. This is how I had it originally but thought to move everything together. I'll rollback.